docs/modules/qbi-helper/overview.md

```diff
@@ -0,0 +1,37 @@
+<%(TOC)%>
+
+# Query-based Interest Helper Module
+
+The Query-based Interest Helper Module provides methods to easily define the `Interest` component used by QBI.
+
+The module includes functionality to:
+
+* Add, replace and clear queries easily from an `Interest` component.
+* Easily define a query's constraints and components to return.
```

Comment:
"components to return", is there a less awkward way of stating this? I can't come up with one off the top of my head.